apps/backoffice/src/pages/new-group-api-key.tsx

```diff
@@ -0,0 +1,171 @@
+import { ButtonCancel, ButtonNext } from "@/components/buttons";
+import { useDialog } from "@/components/dialog-provider";
+import GenerateApiKeyForm from "@/components/groups-api-key/generate-api-key-form";
```

Comment:
As suggested, it would be interesting to use the `CreateUpdateProcess` component (appropriately updated) to create this screen